tests/test_resource_tracker.py

```diff
@@ -0,0 +1,227 @@
+"""Tests for the SemaphoreTracker class"""
+import errno
+import gc
+import io
+import os
+import pytest
+import re
+import signal
+import sys
+import time
+import warnings
+import weakref
+
+from loky import ProcessPoolExecutor
+import loky.backend.resource_tracker as resource_tracker
+from loky.backend.semlock import sem_unlink
+from loky.backend.context import get_context
+
+
+def _resource_unlink(name, rtype):
+    resource_tracker._CLEANUP_FUNCS[rtype](name)
+
+def get_rtracker_pid():
+    resource_tracker.ensure_running()
+    return resource_tracker._resource_tracker._pid
+
+
+class TestSemaphoreTracker:
+
+    @pytest.mark.skipif(sys.platform == "win32",
+                        reason="no resource_tracker on windows")
+    def test_child_retrieves_resource_tracker(self):
+        parent_rtracker_pid = get_rtracker_pid()
+        executor = ProcessPoolExecutor(max_workers=2)
+        child_rtracker_pid = executor.submit(get_rtracker_pid).result()
+
+        # First simple pid retrieval check (see #200)
+        assert child_rtracker_pid == parent_rtracker_pid
+
+        # Register a semaphore in the parent process, and un-register it in the
+        # child process. If the two processes do not share the same
+        # resource_tracker, a cache KeyError should be printed in stderr.
+        import subprocess
+        semlock_name = 'loky-mysemaphore'
+        cmd = '''if 1:
+        import os, sys
+
+        from loky import ProcessPoolExecutor
+        from loky.backend import resource_tracker
+        from loky.backend.semlock import SemLock
```

Comment:
If you do not use the `SemLock` this should be possible to use the same test on `win32` platforms.